### PR TITLE
Update user map for May 2017 #7424

### LINKED ISSUE #7424: Update user map for May 2017
Refer #7215 
Here are the entries to consider
```
Argentina	UNICEN
Argentina	Universidad Nacional de General Sarmiento
Argentina	Universidad Nacional de General Sarmiento
Australia	Federation University
Australia	University of Western Australia
Belgium	GTI Duffel
Belgium	Talentenschool Turnhout
Belgium	University College Artevelde Ghent
Brazil	AEDB
Brazil	State University of Londrina
Brazil	Universidade Católica de Brasília
Canada	Carleton University
Canada	Durham College
Canada	Kelowna Secondary School
Canada	Seneca College
Canada	St. Lawrence College
China	Xi'an Jiaotong-Liverpool University
Denmark	Lyngby Private School
Finland	Teacher Education College JAMK
Finland	The English School
France	ENPC
France	Grenoble Ecole de Management
France	Grenoble Graduate School of Business
France	IESEG School of Management
Greece	University of the Aegean
Hong Kong	Education University of Hong Kong
Ireland	NUI Galway
Malaysia	Monash University Malaysia
Malaysia	Universiti Teknologi Malaysia
Mexico	Centro de Estudios Superiores de Diseño
Mexico	Universidad de Monterrey
Netherlands	Avans University of Applied Science
Netherlands	NHTV Breda University of Applied Sciences
Netherlands	TU Eindhoven
Netherlands	Utrecht University of Applied Sciences
New Zealand	University of Otago
Pakistan	University of Engineering and Technology, Taxila
Peru	UNSA
Philippines	La Salle University
Portugal	Escola Profissional de Almada
Portugal	FEUP
Portugal	Musaic
Saudi Arabia	King Fahd University of Petroleum and Minerals
Singapore	Nanyang Technological University
Singapore	National University of Singapore
Singapore	Singapore Institute of Technology
UK	Edge Hill University
UK	European School of Osteopathy
UK	Northumbria University
UK	Queen's University Belfast
UK	University of Liverpool
UK	University of Warwick
USA	Barrington High School
USA	Binghamton University
USA	Butler County Middle School
USA	California State University, Chico
USA	Duke University
USA	Eastern Kentucky University
USA	EBSCO Information Services
USA	Education Development Center
USA	Elizabethtown College
USA	Endeavour Elementary School
USA	Florence Chapel Middle School
USA	George Mason University
USA	Graland Country Day School
USA	Hult International Business School
USA	Indiana State University
USA	Indiana University
USA	Iowa State University
USA	Kennesaw State University
USA	Kennesaw State University
USA	Lane Community College
USA	Littlestown Area High School
USA	Louisiana State University
USA	Maine South High School
USA	Marshall B. Ketchum University
USA	MIT
USA	Morningside College
USA	NAU
USA	New Bern High School
USA	Norman Regional Health System
USA	Nyack High School
USA	Ohio University
USA	Resurrection University
USA	Saydel Community School
USA	Sonoma State University
USA	Southern Illinois University
USA	Southern Illinois University Edwardsville
USA	Texas Lutheran University
USA	The Bromfield School
USA	Truman State University
USA	Univeristy of Georgia
USA	University of Dallas
USA	University of Louisville
USA	University of Maine
USA	University of Maryland Baltimore County
USA	University of Texas at El Paso
USA	University of Washington
USA	West St. John High School
USA	Woodrow Wilson High School
Vietnam	International University - VNU
```

### PR DESCRIPTION
**Pull request by malenchite**
Fixes #7424 

**Outline of Solution**
Reconciled list of potential new users with existing user list. Researched acronyms and spelling differences to ensure that all new additions were unique.

### CODE REVIEW

File: src/main/webapp/js/userMapData.json [hunk 1]
```diff
@@ -305,7 +320,8 @@
       "Sciences Po",
       "SKEMA Business School",
       "Telecom Paris Tech",
-      "University of Lyon"
+      "University of Lyon",
+      "University of the Aegean"
```
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: This entry should be under "Greece"

File: src/main/webapp/js/userMapData.json [hunk 2]
```diff
@@ -1373,6 +1421,7 @@
       "Southeast Missouri State University",
       "Southern California University of Health Sciences",
       "Southern College of Optometry",
+      "Southern Illinois University",
       "Southern Illinois University Edwardsville",
```
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Confirmed these two are separate universities; no action needed.

File: src/main/webapp/js/userMapData.json [hunk 3]
```diff
@@ -463,7 +480,7 @@
       "Lebanese American University"
     ],
     "Lithuania": [
-      "Šiaulių Dainų progymnasium"
+      "Šiauliu Dainu progymnasium"
```
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: This change should be reverted.

File: src/main/webapp/js/userMapData.json [hunk 4]
```diff
@@ -1,8 +1,10 @@
 {
-  "lastUpdated": "30 April 2017",
+  "lastUpdated": "30 May 2017",
```
Comment (chowyb) [inline src/main/webapp/js/userMapData.json]: Should be "31 May 2017"

**Comment by malenchite** [comment]
I have made the corrections as listed.

**Comment by malenchite** [comment]
Well, I am apparently rustier with git than I expected to be. The test files have been removed. As best I can tell, the only remaining file modification in the branch is the actual user map data.

**Comment by wkurniawan07** [comment]
@damithc apologies, I tried to remove the review request for myself and yet the resulting action was another review request. This features seems to be broken quite badly on GitHub side.

**Comment by whipermr5** [comment]
Merge order: after #7422

**Pull request merged**